src/util/json_expr.cpp

```diff
@@ -259,7 +262,7 @@ json_objectt json(
       if(expr.get(ID_value)==ID_NULL)
         result["data"]=json_stringt("NULL");
     }
-    else if(type.id()==ID_bool)
+    else if(type.id()==ID_bool || type.id()==ID_c_bool)
```

Comment:
For java, this is ok. For c, the situation is a bit tricker (see https://github.com/diffblue/cbmc/pull/357).